conftest.py

```diff
@@ -133,46 +136,88 @@ def noise_model_dict() -> Dict[str, Any]:
 
 
 @pytest.fixture(scope="session")
-def qvm(client: Client):
-    try:
-        qvm = QVMConnection(client=client, random_seed=52)
-        qvm.run(Program(I(0)), [])
-        return qvm
-    except QVMVersionMismatch as e:
-        return pytest.skip("This test requires a different version of the QVM: {}".format(e))
-    except Exception as e:
-        return pytest.skip("This test requires QVM connection: {}".format(e))
+def qvm_connection(client_provider: ClientProvider):
+    qvm = QVMConnection(
+        client_provider=client_provider,
+        random_seed=52,
+    )
+    qvm.run(Program(I(0)), [])
+    return qvm
+
+
+@pytest.fixture(scope="session")
+def qvm_connection_noisy(client_provider: ClientProvider):
```

Comment:
This fixture is for use in tests with mocked responses. So noise doesn't affect results, but we can still validate that noise config is passed downstream correctly.